nms/app/fbcnms-projects/magmalte/app/components/feg/FEGGatewayDialog.js

```diff
@@ -125,10 +123,27 @@ function getDiameterServerConfig(
   };
 }
 
-function getOverwriteAPN(overwrite_apn: ?string): overWriteAPN {
-  return {
-    apn: overwrite_apn || '',
-  };
+function getVirtualApnRules(rules: ?virtual_apn_rules): 
+  ?Array<[string, string]> {
+  if (!rules || rules.length == 0) {
+    return null;
+  }
+  return rules.map((pair) => {
```

Comment:
you can remove above null check and instead do rules?.map and this should return undefined if rules isn't present